documents/entityrelationship/repository.go

```diff
@@ -29,14 +31,58 @@ func newDBRepository(db storage.Repository, docRepo documents.Repository) reposi
 	return r
 }
 
-// Find returns a EntityRelationship based on a entity id and a targetDID
-func (r *repo) Find(entityIdentifier []byte, targetDID identity.DID) (*EntityRelationship, error) {
-	// todo not implemented
-	return nil, nil
+// Find returns the latest (second if revoked) version of an EntityRelationship based on a entity id and a targetDID
+// Note that we assume a case of maximum two versions of an EntityRelationship document
+func (r *repo) FindEntityRelationshipIdentifier(entityIdentifier []byte, ownerDID, targetDID identity.DID) ([]byte, error) {
+	relationships, err := r.db.GetAllByPrefix(string(ownerDID[:]))
+	if err != nil {
+		return nil, err
+	}
+
+	if relationships == nil {
+		return nil, documents.ErrDocumentNotFound
+	}
+
+	for _, r := range relationships {
+		e, ok := r.(*EntityRelationship)
+		if !ok {
+			continue
+		}
+		if bytes.Equal(e.EntityIdentifier, entityIdentifier) {
```

Comment:
Can you inline it like:

```suggestion
		if bytes.Equal(e.EntityIdentifier, entityIdentifier) && targetDID.Equal(*e.TargetIdentity) {
```